src/test/java/com/automatedtest/sample/definitions/CaptureImageSteps.java

```diff
@@ -0,0 +1,46 @@
+package com.automatedtest.sample.definitions;
+
+import com.automatedtest.sample.pages.SettingPage;
+import com.automatedtest.sample.utils.Log;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+public class CaptureImageSteps {
```

Comment:
the test is all about scroll page, not capture image. Capture image is just a method to check the scroll function is working correctly
we should use business logic to naming the steps, not the coding method

Reply:
this Definition class should have only the steps that relate to scrolling, please move other step outside